content/en/docs/concepts/services-networking/service.md

```diff
@@ -1325,15 +1325,15 @@ IP addresses that are no longer used by any Services.
 
 #### IP address ranges for `type: ClusterIP` Services {#service-ip-static-sub-range}
 
-{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.25" state="beta" >}}
 However, there is a problem with this `ClusterIP` allocation strategy, because a user
 can also [choose their own address for the service](#choosing-your-own-ip-address).
 This could result in a conflict if the internal allocator selects the same IP address
 for another Service.
 
-If you enable the `ServiceIPStaticSubrange`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/),
-the allocation strategy divides the `ClusterIP` range into two bands, based on
+The `ServiceIPStaticSubrange`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled by default in v1.25
+and later, using an allocation strategy that divides the `ClusterIP` range into two bands, based on
```

Comment:
Watch out in case this API _doesn't_ graduate - you'll need to fix this wording **and** backport that fix to older docs.

Reply:
Is this a nice-to-have, or are we okay to merge as-is @sftim ?

Reply:
We can merge this as-is - then there's a risk of extra toil to fixup the inaccurate docs.

Reply:
noted